<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>800</width> <height>600</height> </rect> </property>
  <property name="windowTitle"> <string>Sistem Informasi Pernikahan</string> </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>APLIKASI DATA NIKAH</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>800</width> <height>22</height> </rect> </property>
   <widget class="QMenu" name="menuMaster">
    <property name="title"> <string>Menu Master</string> </property>
    <addaction name="actionPria"/>
    <addaction name="actionWanita"/>
    <addaction name="actionDaftar"/>
    <addaction name="actionValidasi"/>
   </widget>
   <addaction name="menuMaster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPria"> <property name="text"> <string>Mempelai Pria</string> </property> </action>
  <action name="actionWanita"> <property name="text"> <string>Mempelai Wanita</string> </property> </action>
  <action name="actionDaftar"> <property name="text"> <string>Pendaftaran</string> </property> </action>
  <action name="actionValidasi"> <property name="text"> <string>Validasi</string> </property> </action>
 </widget>
 <resources/> <connections/>
</ui>
